occult/lib/task/create_models.dart

```diff
@@ -1,25 +1,29 @@
-import 'dart:io';
-
 import 'package:occult/util/tasks.dart';
+import 'package:universal_io/io.dart';
 
 class TCreateModelsProject extends OTaskJob {
   final String app;
 
   TCreateModelsProject(this.app) : super("Create ${app}_models project");
 
   @override
-  Future<void> run() => add(TRun("flutter", [
-        "create",
-        "-t",
-        "package",
-        "--suppress-analytics",
-        "--project-name",
-        "${app}_models",
-        "--no-pub",
-        "--overwrite",
-        "-v",
-        "${app}_models",
-      ]))
+  Future<void> run() => add(
```

Comment:
No no no. Edit TRun dont just hack in ternaries everywhere. this is really dirty, will triple tech debt for future stuff